src/configureWorkspace/configure.ts

```diff
@@ -59,11 +59,15 @@ export interface IPlatformGeneratorInfo {
     genDockerFile: GeneratorFunction,
     genDockerCompose: GeneratorFunction,
     genDockerComposeDebug: GeneratorFunction,
-    defaultPort: string | undefined // '' = defaults to empty but still asks user if they want a port, undefined = don't ask at all
+    defaultPorts: Number[] | undefined // [] = defaults to empty but still asks user if they want a port, undefined = don't ask at all
 }
 
-export function getExposeStatements(port: string): string {
-    return port ? `EXPOSE ${port}` : '';
+export function getExposeStatements(ports: Number[]): string {
+    return ports ? ports.map(port => `EXPOSE ${port}`).join('\n') : '';
```

Comment:
So, this resulted in a unit test failure that exposes an interesting problem. The failure was at configure.test.ts#1231, the ports section was created with `\r\n` (since I ran it on Windows), but the rest of the compose file uses only `\n`, presumably because our templates are all lacking in `\r`. Essentially, we've never put a `\r` into a compose file or Dockerfile before...why start now?